data/iqrafirdose.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "Iqra Firdose",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": " **Open Source** contributor  **Aviyel** ,Hackthoberfest|Web dev Enthisiast | Technical writer | Communities| ",
+    "avatar": "https://github.com/iqrafirdose.png",
+  "tags": ["Developer","Open Source", "Web dev", "Blogger"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/iqra_firdose" },
+    {"icon": "FaLinkedin", "url":"https://www.linkedin.com/in/iqrafirdose/" },
+    { "icon": "FaGithub", "url": "https://github.com/iqrafirdose" },
+    { "icon": "SiHashnode", "url": "https://iqra-firdose.hashnode.dev/" },
```

Comment:
```suggestion
    { "icon": "SiHashnode", "url": "https://iqra-firdose.hashnode.dev/" }
```

Reply:
I think the comma should be there as these are objects in an array which needs to be seperated by comma